### PR TITLE
#71 Schedule search implemented

### PR DESCRIPTION
**Pull request by aayusharora**

### CODE REVIEW

File: js/filter.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+$(document).ready(function(){
```
Comment (joerx) [inline js/filter.js:1]: Please add indentation

File: js/filter.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+$("#filter").on("keyup", function() {
+var g = $(this).val().toLowerCase();
+console.log(g);
```
Comment (joerx) [inline js/filter.js:4]: No console.log

File: schedule/index.html [hunk 3]
```diff
@@ -123,6 +123,9 @@
                 <li><a href="#2016-03-20-exhibition">Exhibition</a></li>
               </ul>
           </ul>
+          <form class="pull-right" style="padding:1%">
```
Comment (joerx) [inline schedule/index.html:126]: Do not edit directly, submit a PR to git@github.com:fossasia/open-event-scraper.git

**Comment by joerx** [comment]
Does not hide the track titles. Does not search speakers.

**Comment by mariobehling** [comment]
Please set up a test site on your github account and share the link for testing.

File: schedule/index.html (hunk 3 above)
Comment (aayusharora) [inline schedule/index.html:126]: @joerx Can you please explain it, why to submit PR for the markup change in 2016.fossasia.org to open-event-scraper.git  ?

Comment (championswimmer) [inline schedule/index.html:126]: @aayusharora  This page is generated by that scraper

Comment (aayusharora) [inline schedule/index.html:126]: @joerx ,generated the PR https://github.com/fossasia/open-event-scraper/pull/17

**Comment by aayusharora** [comment]
@mariobehling ,Are you talking something about this http://aayusharora.github.io/

**Comment by mariobehling** [comment]
@aayusharora Thanks, as described by Joerg. If I type in a speaker in the search the result does not come up. Please make a new PR when you improved on this. Thank you for your work!

**Pull request closed without merging**